lib/config/options/index.ts

```diff
@@ -2733,18 +2733,17 @@ const options: RenovateOptions[] = [
     description:
       'Custom manager to use. Valid only within a `customManagers` object.',
     type: 'string',
-    allowedValues: ['regex'],
+    allowedValues: ['jsonata', 'regex'],
```

Comment:
agree

Reply:
I agree with @RahulGautamSingh

Reply:
Really looking forward to YAML support, I think it would make it possible to write a custom manager for [Hpack](https://github.com/sol/hpack?tab=readme-ov-file#dependencies).

Reply:
So what did we agree here? Is the customer manager type based on:
- The file format it parses (e.g. json, yaml), or
- The language/framework which is used to transform/manipulate the parsed result (e.g. jsonata, regex)?

If in future we have a file named like `filetoparse`, how would a user direct Renovate to do the first level parsing as JSON or YAML?

Reply:
What is the answer?

Reply:
We agreed to base it on the language/framework which is used to transform/manipulate the parsed result. 

If a user has a `filetoparse` then if it's a json/yaml we will convert it internally and parse it to json. But lets say if its another  filetype say, xml and the user wants  to use `jsonata` then they will need to add the support for converting the xml content to json like we do for yaml. 

`jsonata` will be limited imcomparison to regex because it's primarliy made for json structure.